resources/css/bem/counter-box.less

```diff
@@ -5,7 +5,7 @@
   @top: counter-box;
 
   @color-deleted: black;
-  @color-mapper-notes: @beatmap-discussion--color-mapper-note;
+  @color-mapper-notes: var(--beatmapset-discussion-colour--mapper-note);
```

Comment:
also wrong case 👀

Reply:
wonder if the colour for praise should be change to use the variable as well, it's almost the same 👀